src/window.js

```diff
@@ -24,7 +24,34 @@ var ToolboxWindowBase = GObject.registerClass({
         this.hack_toolbox_skeleton = this._createHackToolboxSkeletonOnPath(objectPath);
         this.hackable_proxy = this._maybeGetHackableProxySync();
 
-        this.application.enableFlipBack = false;
+        this._flipBack = new Gio.SimpleAction({
+            name: 'flip-back',
+            parameter_type: new GLib.VariantType('(ss)'),
```

Comment:
I'm not sure we need the parameters anymore, since passing the parameters to the app was necessary to decide what window to route the action too. Now we talk directly to the window already.